### ISSUE TITLE
Add airplay column for servers

### ISSUE THREAD

**Issue by MansM** [basings/selfhosted-music-overview#133]
Add a column that specifies if a server can output to an airplay device (like a receiver etc) over network

**Comment by basings** [comment]
@MansM do you mean airplay from apple? https://www.apple.com/airplay/

**Comment by MansM** [comment]
Yes, like synology's audio station, which can direct audio to airplay devices.

**Comment by basings** [comment]
@MansM k, let's add it. Do you know if any server that's listed supports airplay?

**Comment by BillButter** [comment]
I think Airplay is an app feature, not a server feature.

**Comment by clach04** [comment]
OwnTone - see https://github.com/basings/selfhosted-music-overview/issues/141

**Comment by basings** [comment]
Would it be better to add a column "cast" where one could specify "DLNA" and "airplay" or should it be separate columns?
